Sources/Extensions/URL+Comparable.swift

```diff
@@ -0,0 +1,13 @@
+//  URL+Comparable.swift
+
+import Foundation
+
+extension URL: @retroactive Comparable {
+
+    public static func < (
```

Comment:
imho, such a comparison is not quite unclear.  would remove/replace to more descriptive `sortByLenght`.... because URL("-1.0") URL("10")  what do we compare here?

Reply:
It's a leftover. I'll remove it.